Actions/AL-Go-Helper.ps1

```diff
@@ -1914,17 +1914,17 @@ function GetBaseFolder {
     Param(
         [string] $folder
     )
+    
+    $CurrentDir = Get-Location
```

Comment:
You can use the following pattern here to save one variable and be make it more resilient to potential errors.

```
Push-Location
try {
    doSomething
}
finally {
    Pop-Location
}
```